coderd/database/queries/organizationmembers.sql

```diff
@@ -89,6 +89,12 @@ WHERE
 			organization_id = @organization_id
 		ELSE true
 	END
+  -- Filter by system type
+	AND CASE
+		WHEN @include_system::bool THEN TRUE
+		ELSE
+			is_system = false
+	END
```

Comment:
nit for readability:
```suggestion
  -- Conditionally include system users based on @include_system flag
	AND (@include_system OR is_system = false)
```

Reply:
done!